.prettierignore

```diff
@@ -0,0 +1,5 @@
+node_modules
+*.log
```

Comment:
These files are not `.js`, `.md` or `.json`, and you run `prettier` only on files with extensions mentioned above:
https://github.com/metarhia/metaschema/blob/2256c955ad58d93374f15ffd0b104b7d5e650e0d/package.json#L36-L37
So there is no point in adding this line.
```suggestion
```